e2e/transform/transform-environment/__tests__/add.test.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+it('should add two numbers', () => {
+  expect(1 + 1).toBe(2);
```

Comment:
Maybe we should use something from the env here just to be sure that it was properly loaded